src/oscar/apps/analytics/abstract_models.py

```diff
@@ -101,11 +102,12 @@ class AbstractUserSearch(models.Model):
         on_delete=models.CASCADE,
         verbose_name=_("User"))
     query = models.CharField(_("Search term"), max_length=255, db_index=True)
-    date_created = models.DateTimeField(_("Date Created"), auto_now_add=True)
+    date_created = models.DateTimeField(_("Date Created"), auto_now_add=True, db_index=True)
 
     class Meta:
         abstract = True
         app_label = 'analytics'
+        ordering = ['-date_created']
```

Comment:
Think you may have missed [my previous comment](https://github.com/django-oscar/django-oscar/pull/3364#discussion_r426118907) on this?

Reply:
The previous comment was for the basket app. Should I change the ordering here to '-pk' as well?

Reply:
Yes please.

Reply:
Could you please also delete and regenerate the migrations, so that we're left with just one new migration per app.

Reply:
Of course!....On it...